de.ifw_dresden.nagstamon.yml

```diff
@@ -0,0 +1,145 @@
+app-id: de.ifw_dresden.nagstamon
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: nagstamon
+finish-args:
+  - --socket=wayland  
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=~/.nagstamon:create
```

Comment:
It would be nice if you could report an issue about using xdg-dirs instead of this would be great :)

Reply:
Or at least use `--persist=.nagstamon` instead of --filesystem. It will create the .nagstamon directory inside the Flatpak's directory in ~/.var/app. ;-)

Reply:
mmm I didn't know about that "--persist" option. Just had a look to the documentation and looks preferable. I will test it locally and push the changes if everything is ok. Thanks you.